Fix style violations (#382)
obsidianmd/obsidian-docs: en/How to/Basic note taking.md

If you've just opened Obsidian, you'll see two documents here. The one in the left pane is a text file, which you can edit using Markdown syntax. The one in the right pane is a _Preview_ of the same text file if you were to export it as HTML, or as a PDF document. Note that some things look different in Preview, and, most obviously, links aren't in brackets. To follow a link, click on it in the Preview pane. You can follow links in the Editor pane as well, by holding down `Ctrl/Cmd` and clicking on the link. If you want to go back to the page you were on before, use the shortcut `Ctrl/Cmd-alt-LeftArrow`

> Alternatively, the notes that link to the one you're currently on is be shown in the **Backlinks Panel** on the right (find more about Backlinks [[Backlinks|here.]])

Reviewer: ```suggestion
Alternatively, the notes that link to the one you're currently on are shown in the **Backlinks Panel** on the right (find more about Backlinks [[Backlinks|here.]])
```

If you get lost at any time, or just want to explore what else Obsidian can do, `Ctrl/Cmd-P` brings up the Command Palette. You can type commands there, like "Navigate back", and you'll see options filter to a short list, along with hotkey shortcuts if they have them.

At base, Obsidian is a tool for working with a bunch of Markdown files that are stored on your computer. If you're not familiar with Markdown, it's just plain text with some formatting that makes it powerful (for more on that, see [[Format your notes|Markdown]]) . As a result, if you want to use Obsidian for keeping regular notes on just about anything, it will work great.

Let's say you want to write a note about your favorite band. Just type the name of the note surrounded by two square brackets, like `[[My Favorite Band]]`

> Try it here (make sure you're typing in the Editor pane):

Reviewer: ```suggestion
```
